discord/opus.py

```diff
@@ -57,6 +64,15 @@ class SignalCtl(TypedDict):
     music: int
 
 
+class EncoderKwargs(TypedDict, total=False):
+    application: APPLICATION_CTL
+    bitrate: int
+    fec: bool
```

Comment:
This one's technically a shortcut too but I'm unsure if it's better off in long form or not. No strong opinion, just noting it.